CMakeLists.txt

```diff
@@ -571,23 +589,29 @@ if (NOT LWS_WITHOUT_TESTAPPS)
 		source_group("Headers Private"   FILES ${TEST_HDR})
 		source_group("Sources"   FILES ${TEST_SRCS})
 		add_executable(${TEST_NAME} ${TEST_SRCS} ${TEST_HDR})
-		
+
 		if (LWS_LINK_TESTAPPS_DYNAMIC)
+			if (NOT LWS_WITH_SHARED)
+				message(FATAL_ERROR "Build of shared library is disabled, can't link the test apps to it")
```

Comment:
change to `message(FATAL_ERROR "Build of shared library is disabled, can't link the test apps to it. LWS_LINK_TESTAPPS_DYNAMIC must be combined with LWS_WITH_SHARED")`